pkg/apis/pipeline/v1beta1/pipelinerun_validation_test.go

```diff
@@ -1557,3 +1557,205 @@ func TestPipelineRunWithTimeout_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestPipelineRunSpecBetaFeatures(t *testing.T) {
```

Comment:
NIT: please add doc string explaining what is being tested

Reply:
Thanks Priti. Added:)